### PR TITLE
test-case: improve Chrome OS compatibility

### PR DESCRIPTION
**Pull request by aiChaoSONG**
These two patches make check-playback/capture test cases can run on Chrome smoothly. suppose sof-test is downloaded to /usr/local, and other dependencies are installed with [chromebrew](http://skycocker.github.io/chromebrew)

### CODE REVIEW

**Review by paulstelian97 — APPROVED** [review]

**Review by plbossart — DISMISSED** [review]
Thanks @aiChaoSONG.
maybe that's a stretch but does this help with Fedora compatibility as well? What's missing now?

**Comment by aiChaoSONG** [comment]
> Thanks @aiChaoSONG.
> maybe that's a stretch but does this help with Fedora compatibility as well? What's missing now?

Some commands in sof-test need root privilege, so they are executed as `sudo cmd`, for example, sudo sof-logger. On Ubuntu, we add current user to sudo group, then we don't need to input password when run cmd with sudo.

The same thing needs to be done for Fedora and Chrome, but there is not a sudo group on these two distros as Ubuntu, what I do is to
- add current user to audio group: `sudo usermod --append --groups audio $USER`
- config audio group in `visudo`, add `audio  ALL=(ALL) NOPASSWD: ALL`

After this, you can run sof-test on fedora, just as you run it on ubuntu.

**Review by marc-hb — DISMISSED** [review]

**Review by marc-hb — APPROVED** [review]
`[  833.106967] kernel: IO error in rt700_jack_detect_handler, ret -16` in 
https://sof-ci.01.org/softestpr/PR684/build716/devicetest/?model=CML_RVP_SDW&testcase=check-suspend-resume-5 is unrelated https://github.com/thesofproject/linux/issues/2943

File: case-lib/lib.sh [hunk 1]
```diff
@@ -335,7 +335,7 @@ is_sof_used()
 # a wrapper to journalctl with required style
 journalctl_cmd()
 {
-   journalctl -k -q --no-pager --utc --output=short-monotonic \
+   sudo journalctl -k -q --no-pager --utc --output=short-monotonic \
```
Comment (marc-hb) [inline case-lib/lib.sh:338]: @aiChaoSONG I just had a look a recent ChromeOS image (less than a couple weeks old) and I found zero trace of `journalctl`, only `rsyslogd`... can you explain?
- https://github.com/thesofproject/sof-test/issues/576#issuecomment-1295641285

Comment (aiChaoSONG) [reply case-lib/lib.sh:338]: At the time I tried sof-test on chrome, there was `journalctl`. I have no idea why they remove journalctl in later chrome image, but there is a new tool for log, `craslog`, which works just like `journalctl`.

Comment (marc-hb) [reply case-lib/lib.sh:338]: You're right: there was journalctl but only for a short time. With your help I found a bit more info, see #576

**Pull request merged**